homeassistant/components/sensor/miflora.py

```diff
@@ -51,14 +49,15 @@
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
     vol.Optional(CONF_MEDIAN, default=DEFAULT_MEDIAN): cv.positive_int,
     vol.Optional(CONF_FORCE_UPDATE, default=DEFAULT_FORCE_UPDATE): cv.boolean,
-    vol.Optional(CONF_TIMEOUT, default=DEFAULT_TIMEOUT): cv.positive_int,
-    vol.Optional(CONF_RETRIES, default=DEFAULT_RETRIES): cv.positive_int,
-    vol.Optional(CONF_CACHE, default=DEFAULT_UPDATE_INTERVAL): cv.positive_int,
     vol.Optional(CONF_ADAPTER, default=DEFAULT_ADAPTER): cv.string,
+    vol.Optional(CONF_SCAN_INTERVAL, default=DEFAULT_SCAN_INTERVAL):
+        cv.time_period,
 })
 
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
+@asyncio.coroutine
```

Comment:
Use Python 3.5 async syntax,`async def`, instead of the coroutine decorator.

Reply:
Okay, so just copy-paste from `roomba.py` doesn't mean review will be successful :smile:. Fixed.